page-laws.php

```diff
@@ -0,0 +1,210 @@
+<?php
+// dbug
```

Comment:
@chris-aeviator no template header here?? It should be something like
```
<?php
/*
Template Name: Laws page
*/
?>
```
See map-explorer.php
https://github.com/OpenDevelopmentMekong/opendev-wptheme/blob/master/page-map-explorer.php#L2